config/localTemplate.go

```diff
@@ -231,7 +231,20 @@ type Local struct {
 	// TxBacklogReservedCapacityPerPeer determines how much dedicated serving capacity the TxBacklog gives each peer
 	TxBacklogReservedCapacityPerPeer int `version[27]:"20"`
 
-	// EnableTxBacklogRateLimiting controls if a rate limiter and congestion manager shouild be attached to the tx backlog enqueue process
+	// TxBacklogAppTxRateLimiterMaxSize denotes a max size for the tx rate limiter
+	// calculated as "a thousand apps on a network of thousand of peers"
+	TxBacklogAppTxRateLimiterMaxSize int `version[32]:"1048576"`
```

Comment:
For naming consistency with the overall knob, should these related settings all start with `AppTxBacklog` as their prefix? Make it easy to distinguish the TxRateLimiter settings from the the app tx rate limiter

Reply:
idk, TxBacklogXXX vs TxBacklogAppXXX looks consistent